app/models/metric/ci_mixin/capture.rb

```diff
@@ -168,6 +166,10 @@ def perf_capture_callback(task_ids, _status, _message, _result)
     end
   end
 
+  def perf_capture_states(targets)
+    Benchmark.realtime_block(:capture_state) { VimPerformanceState.capture(targets) }
```

Comment:
this is called from
```ruby
perf_capture_states(metrics_capture.target)
```

which is deceiving because for some metrics_capture, `target` is plural.
And tbh, going forward, more and more providers will have multiple targets (the premise of the perf_capture initiative)